code/modules/modular_computers/file_system/programs/sm_monitor.dm

```diff
@@ -128,40 +128,16 @@
 		if(!air)
 			active = null
 			return
-
-		data["active"] = TRUE
-		data["SM_integrity"] = active.get_integrity()
-		data["SM_power"] = active.power
-		data["SM_radiation"] = active.last_rads
-		data["SM_ambienttemp"] = air.return_temperature()
-		data["SM_ambientpressure"] = air.return_pressure()
-		data["SM_bad_moles_amount"] = MOLE_PENALTY_THRESHOLD / active.gasefficency
-		data["SM_moles"] = 0
-
-		var/list/gasdata = list()
-
-		if(air.total_moles())
-			data["SM_moles"] = air.total_moles()
-			for(var/gasid in air.get_gases())
-				gasdata.Add(list(list(
-				"name"= GLOB.gas_data.names[gasid],
-				"amount" = round(100*air.get_moles(gasid)/air.total_moles(),0.01))))
-
```

Comment:
Does the normal SM console still work properly? If so this should be good to go

Reply:
Yes it does, see testing screenshot